docs/how_to/install_puppet_enterprise.md

```diff
@@ -0,0 +1,107 @@
+# How-To Install Puppet Enterprise
+
+Welcome to the choose your own adventure of installing Puppet Enterprise (PE)!
+This doc will guide you through the process beaker's DSL helpers use to install
+PE for acceptance testing.
+
+Note that this is not a complete documentation of the process, but a general
+overview. There will be specific hiccups for particular platforms and special
+cases. These are not included at this point. The idea is that as we come upon
+them, we will now have a place to document those details, so that we can
+over time bring this to 100% completeness.
+
+# First Things First: What Do You Want to Install?
```

Comment:
Yeah, `install_puppet_agent_pe_promoted_repo_on` is a part of PE installation. A large part of this doc is to guide people to potentially using this method rather than the typical `install_pe` method that people use if they want to specifically just install the same agents that PE would use.